When running an install, we want the log to be displayed.

diff --git a/lib/bb/ui/knotty.py b/lib/bb/ui/knotty.py
index 73327b5..58df197 100644
--- a/lib/bb/ui/knotty.py
+++ b/lib/bb/ui/knotty.py
@@ -276,7 +276,7 @@ class StdinMgr:
 
 class RtLogLevel:
     def __init__(self, handler, logfilter, mlt, tf):
-        self.displaytail = False
+        self.displaytail = True
         self.displayLogLocations = False
         self.handler = handler
         self.logfilter = logfilter
